src/ngraph_builder.cc

```diff
@@ -1161,6 +1211,9 @@ static Status TranslateConv2DBackpropFilterOp(
                       static_cast<unsigned int>(tf_filter_sizes[0]),
                       static_cast<unsigned int>(tf_filter_sizes[1])};
   BatchToNGraph(is_nhwc, ng_output_delta);
+  if (is_nhwc) {
```

Comment:
Q: We have already taken care of the case for "is_nhwc" here: https://github.com/NervanaSystems/ngraph-tf/pull/413/files?utf8=%E2%9C%93&diff=unified&w=1#diff-148e2a1788305495ad1a483e17b53eaaR1203
Why are we doing it again?

Reply:
They (ng_data_batch and ng_output_delta) are different data flows/ inputs so both of them will be entry nodes.